src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  --blue-color: #00acdc;
```

Comment:
Set variables for root. It's common best practice:
 https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties
```
:root {
  --blue-color: #00acdc;
}
```